src/org/camunda/latera/kotlin/bss/connectors/JasperReport.kt

```diff
@@ -1,7 +1,69 @@
 package org.camunda.latera.bss.connectors
 
-class JasperReport {
-  fun testFunction() {
-    throw Exception("Hi There!")
+import org.camunda.latera.bss.HttpClient.HttpProcessor
+import io.ktor.client.request.*
+import io.ktor.client.HttpClient
+import io.ktor.client.features.json.GsonSerializer
+import io.ktor.client.statement.HttpResponse
+import io.ktor.client.features.json.JsonFeature
+import io.ktor.http.HttpMethod
+import io.ktor.http.ContentType
+import io.ktor.http.HttpStatusCode
+import io.ktor.http.contentType
+import io.ktor.http.headersOf
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import kotlinx.coroutines.*
+
+data class export(val id: String, val status: String)
+
+data class executeReportParams(val reportUnitUri: String,
+                               val parameters: Map<String, Any>,
+                               val outputFormat: String = "pdf",
+                               val freshData: Boolean = false,
+                               val saveDataSnapshot: Boolean = false,
+                               val interactive: Boolean = true,
+                               val allowInlineScripts: Boolean = true,
+                               val ignorePagination: Boolean? = null,
+                               val pages: Int? = null,
+                               val async: Boolean = false,
+                               val transformerKey: String? = null,
+                               val attachmentsPrefix: String = "attachments",
+                               val baseUrl: String? = null)
+
+data class executeReportResponse(val currentPage: Int,
+                                 val exports: Array<export>,
+                                 val reportURI: String,
+                                 val requestId: String,
+                                 val status: String)
+
+class JasperReport(val url: String, val user: String, val password: String) {
+  fun executeReport(params: executeReportParams) : executeReportResponse {
+    val executeReportUrl = "${this.url}/reportExecutions"
```

Comment:
We can have trailing slash in `this.url`, so let's use any URL builder to avoid duplicated slash.